scipy/spatial/transform/rotation.py

```diff
@@ -1460,3 +1462,54 @@ def __getitem__(self, indexer):
                [ 0.57735027,  0.57735027, -0.57735027,  0.        ]])
         """
         return self.__class__(self._quat[indexer], normalized=True)
+
+    @classmethod
+    def random(cls, num=None, random_state=None):
+        """Generate uniformly distributed rotations.
+
+        Parameters
+        ----------
+        num : None or int
+            Number of random rotations to generate. If None, then a single
+            rotation is generated. Default is None.
+        random_state : None, int or `numpy.random.RandomState` object
+            If RadomState object, used for generator state. If integer, used
```

Comment:
Typo `RandomState`. 

Also you can simply write `None, optional` as the type and explain the possibilities in the description. E.g., 

    It accepts an `int` as a seed for the random generator or a RandomState object. 

Then you can explain what else this keyword accepts.

Reply:
Bikeshedding nitpick: with optional arguments, you should describe the type as what you intend people to actually pass when they pass something rather than the type of the default when they pass nothing. Personally, I'd write this as `random_state : RandomState or int, optional`

Reply:
Or keep the current wording and tack on `, optional`. E.g. https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/gaussian_process/gpr.py#L108